Assets/Scripts/UI/TwitchComponentHandle.cs

```diff
@@ -563,8 +563,6 @@ public IEnumerator OnMessageReceived(string userNickName, string userColor, stri
 						return null;
 					}
 				}
-
-				if (messageOut == null) ircConnection.SendMessage(TwitchPlaySettings.data.TwitchPlaysDisabled, userNickName);
```

Comment:
I do agree that this line is incorrect, but we do need some way to message users who aren't authorized.

Reply:
And that happens later on in the code.  Besides that, those who aren't authorized will never reach that line of code anyways, and those who are, and don't send one of the above commands that sets messageOut, will ALWAYS hit that line.